Date: Nov 12, 2014 2:13 AM

From: b74e22dc341b0b64ec5772955a06f30068cf11da Nov 15, 2014 5:23 AM

Subject: [PATCH] RDKSYSINTSW-1315: Use common distro for both RDK and RDK-B

        A client for automaticly updating your dynamic hostname parameters.

Source: GITHUB


Upstream-Status: Submitted [ http://cgit.openembedded.org/meta-openembedded/commit/meta-networking/recipes-connectivity/ez-ipupdate/files/cache_file.c.patch?h=sumo&id=cf51ee759339dba2364aa570665d725f11d0b8a7 ]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
---

--- ez-ipupdate-3.0.11b7/Makefile.am.orig	2014-07-02 13:47:50.758034983 -0600
+++ ez-ipupdate-3.0.11b7/Makefile.am	2014-07-02 13:48:38.406034650 -0600
@@ -1,7 +1,7 @@
 
 bin_PROGRAMS = ez-ipupdate
-ez_ipupdate_SOURCES = ez-ipupdate.c conf_file.c conf_file.h md5.c md5.h cache_file.c cache_file.h error.h pid_file.c pid_file.h dprintf.h @EXTRASRC@
-ez_ipupdate_LDADD = @EXTRAOBJ@
+ez_ipupdate_SOURCES = ez-ipupdate.c conf_file.c conf_file.h md5.c md5.h cache_file.c cache_file.h error.h pid_file.c pid_file.h dprintf.h
+ez_ipupdate_LDADD =
 
 EXTRA_DIST = getpass.c ez-ipupdate.lsm example.conf example-pgpow.conf example-dhs.conf example-dyndns.conf example-ods.conf example-tzo.conf example-gnudip.conf example-easydns.conf example-justlinux.conf example-dyns.conf CHANGELOG mkbinary example-heipv6tb.conf
 
